addons/l10n_pe/data/template/account.tax-pe.csv

```diff
@@ -1,61 +1,61 @@
+"","","","","","","","","","","","","tax","refund","chart40111","","",""
+"purchase_tax_igv_18_included","VAT 18% Incl.","VAT 18% Included Purchase","IGV","1000","S","18.0","purchase","1","1","tax_group_igv","True","base","invoice","","","IGV 18% Incl.","IGV 18% Incluido en Precio Compra"
+"","","","","","","","","","","","","tax","invoice","chart40111","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40111","","",""
+"purchase_tax_igv_18g_ng","VAT 18% G NG","VAT 18% Taxed and Untaxed Purchase","IGV G NG","1000","S","18.0","purchase","1","1","tax_group_igv_g_ng","True","base","invoice","","","IGV 18% G NG","IGV 18% Gravadas y No Gravadas"
+"","","","","","","","","","","","","tax","invoice","chart40117","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40117","","",""
+"purchase_tax_igv_18_ng","VAT 18% NG","VAT 18% Untaxed Purchase","IGV NG","1000","S","18.0","purchase","1","1","tax_group_igv_ng","True","base","invoice","","","IGV 18% NG","18% No Gravadas"
+"","","","","","","","","","","","","tax","invoice","chart40116","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40116","","",""
+"purchase_tax_exp_0","0% Exp","0% Exportation Purchase","EXP","9995","S","18.0","purchase","1","1","tax_group_exp","True","base","invoice","","","0% Exp","0% Exp"
+"","","","","","","","","","","","","tax","invoice","chart40115","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40115","","",""
+"purchase_tax_exo","0% Exo","0% Exonerated Purchase","EXO","9997","E","0.0","purchase","1","","tax_group_exo","","base","invoice","","","0% Exo","0% Exonerado"
+"","","","","","","","","","","","","tax","invoice","chart40111","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40111","","",""
+"purchase_tax_ina","0% Ina","0% Inaffected Purchase","INA","9998","Z","0.0","purchase","1","","tax_group_ina","","base","invoice","","","0% Ina","0% Inafectado"
+"","","","","","","","","","","","","tax","invoice","chart40111","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40111","","",""
+"purchase_tax_gra","0% Gra","0% Free Purchase","GRA","9996","E","0.0","purchase","1","","tax_group_gra","","base","invoice","","","","0% Gratis"
+"","","","","","","","","","","","","tax","invoice","chart40111","","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","chart40111","","",""
+"sale_tax_exp","0% Exp","0% Exportation Sale","EXP","9995","S","0.0","sale","1","1","tax_group_exp","","base","invoice","","0","","0% EXP"
```

Comment:
And just a small one here @aliyatastemirova the part "0% EXP" at the end of the line should be "0% Exp". 

Thanks!

Fernando